api/src/main/java/org/openmrs/api/db/hibernate/HibernateOrderDAO.java

```diff
@@ -1,4 +1,4 @@
-/**
+	/**
```

Comment:
I just tried running that test in my PC and it compiled without any errors , now i'm kinda clueless regarding the error out here.Any ideas?